pio-tools/http-uploader.py

```diff
@@ -10,4 +10,4 @@
 # pio >= 4.0.0
 env.Replace(UPLOADER=os.path.join("pio-tools", "espupload.py"))
 env.Replace(UPLOADERFLAGS="")
-env.Replace(UPLOADCMD="$UPLOADER -u $UPLOAD_PORT -f $SOURCES")
+env.Replace(UPLOADCMD="python $UPLOADER -u $UPLOAD_PORT -f $SOURCES")
```

Comment:
Better using `$PYTHONEXE` instead of `python`

Reply:
Thx. Works great